Add wiki tutorials submodules

nikiOstrov opened the pull request.

nikiOstrov committed:
adding git steps for dispatching wiki tutorials

nikiOstrov committed:
added wiki tutorials sub module

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/devonfw/devonfw.github.io?pullRequest=249) <br/>All committers have signed the CLA.

SchettlerKoehler commented on .github/workflows/dispatch-event.yml:
```diff
@@ -36,6 +36,13 @@ jobs:
           cd ..
           git add katacoda-scenarios
+          cd wiki-tutorials/
+          git checkout master
```
The default branch of that repo is main not master

SchettlerKoehler commented on .gitmodules:
```diff
@@ -2,6 +2,10 @@
 	path = katacoda-scenarios
 	url = https://github.com/devonfw/katacoda-scenarios.git
```
Please change this to https://github.com/devonfw-tutorials/katacoda-scenarios.git
This is the old URL of the repo and is only working because of github redirection magic

SchettlerKoehler commented on .gitmodules:
```diff
@@ -2,6 +2,10 @@
+[submodule "wiki-tutorials"]
+	path = wiki-tutorials
+	url = https://github.com/devonfw-tutorials/wiki-tutorials.git
+	branch = master
```
See my comment above on the branch name

nikiOstrov committed:
changed base branch name to main

changed the name of the referred branch from master -> main

nikiOstrov committed:
chnaged to old katacoda scenarios url

nikiOstrov committed:
changed directory name to main

changed the directory name in the wiki-tutorials repository from master to main

denise-khuu commented on .github/workflows/dispatch-event.yml:
```diff
@@ -36,6 +36,13 @@ jobs:
+          git checkout main
+          git fetch
+          git branch
+          git merge origin/master
```
origin/main

nikiOstrov commented:
Branch has been accidentally deleted, cannot make last changes - wil start new PR
